features/promotion/receiving_discount_based_on_number_of_items_from_specific_taxon.feature

```diff
@@ -0,0 +1,55 @@
+@promotion
+Feature: Receiving discount based on number of products from specific taxon
+    In order to pay less while buying required number of goods from promoted taxon
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store classifies its products as "T-Shirts" and "Mugs"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
```

Comment:
I'm wondering if it wouldn't be better to rename this step to: `And this product belongs to "T-Shirts"`
wdyt?